MovieGallery/Model/Shows.cs

```diff
@@ -80,10 +80,13 @@ internal class ParseStringConverter : JsonConverter
 
         public override object ReadJson(JsonReader reader, Type t, object existingValue, JsonSerializer serializer)
         {
-            if (reader.TokenType == JsonToken.Null) return null;
+            if (reader.TokenType == JsonToken.Null)
+            {
+                return null;
+            }
+
```

Comment:
Added brackets after the if statement, as its best practice to use the syntax.
https://www.w3schools.com/cs/cs_conditions.asp